packages/blob/README.md

```diff
@@ -131,38 +230,41 @@ async function list(options?: {
 
 ### handleBlobUpload(options)
 
-Handles the requests to generate a client token and respond to the upload completed event. This is useful when [uploading directly from browsers](#uploading-directly-from-browsers) to circumvent the 4MB limitation of going through a Vercel-hosted route.
+Handles the requests to generate a client token and respond to the upload completed event. This is useful when [uploading from browsers](#browser-upload) to circumvent the 4MB limitation of going through a Vercel-hosted route.
 
 ```ts
 async function handleBlobUpload(options?: {
-  token?: string;
-  request?: IncomingMessage | Request;
+  token?: string; // default to process.env.BLOB_READ_WRITE_TOKEN
+  request: IncomingMessage | Request;
   onBeforeGenerateToken: (pathname: string) => Promise<{
-    allowedContentTypes?: string[];
-    maximumSizeInBytes?: number;
-    validUntil?: number; // timestamp in ms, by default 30s
+    allowedContentTypes?: string[]; // optional, defaults to no restriction
+    maximumSizeInBytes?: number; // optional, defaults and maximum is 500MB (524,288,000 bytes)
```

Comment:
where does this limit come from? CF? 
this may change depending on the user plan, right?

Reply:
Yes this comes from Cloudflare: https://developers.cloudflare.com/workers/platform/limits/#request-limits

Reply:
ok, in case we apply lower limits for hobby users we can review this comment later on